.github/workflows/daily-update-functions.yaml

```diff
@@ -0,0 +1,53 @@
+name: DailyUpdateFunctions
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 15 * * *'
+env:
+  go-version: 1.16
+  project: neco-dev
```

Comment:
This value is not used. Please delete it.

Reply:
thx! I forgot to delete it